tests/unit/store/getters.spec.ts

```diff
@@ -3,44 +3,46 @@ import getters from '@/store/getters';
 import QueryRepresentation from '@/sparql/QueryRepresentation';
 import PropertyValueRelation from '@/data-model/PropertyValueRelation';
 
-const simpleRootState: RootState = {
-	conditionRows: [ {
-		valueData: { value: 'foo', valueError: null },
-		propertyData: {
-			id: 'P123',
-			label: 'abc',
-			datatype: 'string',
-			isPropertySet: true,
-			propertyError: null,
-		},
-		propertyValueRelationData: { value: PropertyValueRelation.Matching },
-		conditionId: '0.123',
-		subclasses: false,
-		negate: false,
-	} ],
-	limit: 0,
-	useLimit: false,
-	omitLabels: true,
-	errors: [],
-};
-
 describe( 'getters', () => {
+	function getFreshRootState(): RootState {
+		const simpleRootState: RootState = {
+			conditionRows: [ {
+				valueData: { value: 'foo', valueError: null },
+				propertyData: {
+					id: 'P123',
+					label: 'abc',
+					datatype: 'string',
+					isPropertySet: true,
+					propertyError: null,
+				},
+				propertyValueRelationData: { value: PropertyValueRelation.Matching },
+				conditionId: '0.123',
+				subclasses: false,
+				negate: false,
+			} ],
+			limit: 0,
+			useLimit: false,
+			omitLabels: true,
+			errors: [],
+		};
+		return JSON.parse( JSON.stringify( simpleRootState ) );
```

Comment:
Also that JSON.parse and JSON.stringify shouldn't be needed. Just return the object :)